controllers/os-suse-jeos/example/controller-registration-local.yaml

```diff
@@ -0,0 +1,16 @@
+apiVersion: core.gardener.cloud/v1alpha1
```

Comment:
This is the one required for registering the controller locally. The helm chart is "dummy" but allows the controller to be properly "installed" by gardener.

Reply:
OK, I got confused because you provide an image tag. Can you then remove

```yaml
      values:
        image:
          tag: 0.1.0-dev
```

It seems not needed, right?

Reply:
Fixed.